tests/kafkatest/services/zookeeper.py

```diff
@@ -158,3 +157,9 @@ def query(self, path, chroot=None):
                 if match is not None:
                     result = match.groups()[0]
         return result
+
+    def java_class_name(self):
+        return "org.apache.zookeeper.server.quorum.QuorumPeerMain"
+
+    def java_query_class_name(self):
```

Comment:
Might help to add a brief comment on why this is needed in addition to `java_class_name`?